dts/bindings/cpu/altera,nios2f.yaml

```diff
@@ -3,7 +3,7 @@
 
 description: Altera NIOS-2F CPU
 
-compatible: "altera,nios2f"
+compatible: "altr,nios2f"
 
```

Comment:
I'm late to the party, but this file (and altera,jtag-uart.yaml) need to be renamed too.

Reply:
> I'm late to the party, but this file (and altera,jtag-uart.yaml) need to be renamed too.

I'll send a followup PR for the file rename.